benchmarks/fast_serializable_check.py

```diff
@@ -0,0 +1,46 @@
+import time
+
+import arctic.serialization.numpy_records as anr
+from tests.unit.serialization.serialization_test_data import _mixed_test_data as input_test_data
+
+df_serializer = anr.DataFrameSerializer()
+
+
+def _bench(rounds, input_df, fast):
+    fast = bool(fast)
+    anr.set_fast_check_df_serializable(fast)
+    start = time.time()
+    for i in range(rounds):
+        df_serializer.can_convert_to_records_without_objects(input_df, 'symA')
+    print("Time per iteration (fast={}): {}".format(fast, (time.time() - start)/rounds))
```

Comment:
Curious: does python suffer any of the same issues as java for microbenchmarking?

Reply:
JVM definitely has many more optimisations in place, which are affected by micro-benchmarking, and make it tough to deduce safe results for the real-world performance, of the whole app.

Python in this sense is not suffering from the same issues, but still, micro-benchmarking is used more for runtime trends of small code fragments, and shouldn't be used in a generalized manner for the whole app performance.